ouroboros-consensus/test-storage/Test/Ouroboros/Storage/LedgerDB/OnDisk.hs

```diff
@@ -941,15 +944,18 @@ prop_sequential :: LUT t => Proxy t -> MemPolicy -> QC.Property
 prop_sequential p memPolicy =
     QC.collect (tagMemPolicy memPolicy) $
       forAllCommands (sm memPolicy (dbUnused p)) Nothing $ \cmds ->
-        QC.monadicIO (propCmds memPolicy cmds)
+        QC.monadicIO (propCmds k memPolicy cmds)
+  where
+    k = SecurityParam $ memPolicyMaxRollback memPolicy
 
 -- Ideally we'd like to use @SimM s@ instead of IO, but unfortunately
 -- QSM requires monads that implement MonadIO.
 propCmds :: LUT t
-         => MemPolicy
+         => SecurityParam
+         -> MemPolicy
          -> QSM.Commands (At (Cmd t)) (At (Resp t))
          -> QC.PropertyM IO ()
-propCmds memPolicy cmds = do
+propCmds k memPolicy cmds = do
```

Comment:
Must it be that `k = memPolicyMaxRollback memPolicy`? In that case, making it a parameter introduces the risk that somebody passes in the wrong `k`.

Reply:
Fair point. Don't actually need this anymore. Will change this back.